src/libs/TransactionUtils.ts

```diff
@@ -450,12 +452,13 @@ function getCreated(transaction: OnyxEntry<Transaction>, dateFormat: string = CO
 /**
  * Returns the translation key to use for the header title
  */
-function getHeaderTitleTranslationKey(transaction: Transaction): string {
-    const headerTitles = {
+function getHeaderTitleTranslationKey(transaction: OnyxEntry<Transaction>): TranslationPaths {
+    const headerTitles: Record<string, TranslationPaths> = {
```

Comment:
```suggestion
    const headerTitles: Record<ValueOf<typeof CONST.IOU.REQUEST_TYPE>, TranslationPaths> = {
```
But as I mentioned, let's create a separate type for `ValueOf<typeof CONST.IOU.REQUEST_TYPE>`